integration-tests/bot/jail_test.py

```diff
@@ -61,6 +61,10 @@ def create_addresses(self):
     def unjail(self,name, passphrase, address):
         try:
             return self.rpc.staking.unjail(address, name)
+        except KeyboardInterrupt as ex:
```

Comment:
We should just let these exceptions propagate, if we needs to write the logs, we can write it as:
```
except:
    log()
    raise
```

Reply:
no, this is polling , not panic state